test/e2e/specs/all-features-confirmation/errors/errors.spec.js

```diff
@@ -9,7 +9,7 @@ var testParams = {
     deletionErrTextBooking : "This entry cannot be deleted as it is still referenced from the booking table. All dependent entries must be removed before this item can be deleted. If you have trouble removing dependencies please contact the site administrator.\n\nClick OK to go to the Accommodations.\nShow Error Details",
     deletionErrTextAccommodationImg : "This entry cannot be deleted as it is still referenced from the accommodation_image table. All dependent entries must be removed before this item can be deleted.\n\nClick OK to go to the Accommodations.\nShow Error Details",
     uniqueConstraint : "Error The entry cannot be created/updated. Please use a different ID for this record.",
-    recordNotFoundModalText : "No matching record found for the given filter or facet.\n\nClick OK to show the list of all records.\nShow Error Details",
+    recordNotFoundModalText : "The record does not exist or may be hidden. If you continue to face this issue, please contact the system administrator.\n\nClick OK to show the list of all records.\nShow Error Details",
```

Comment:
This isn't testing the "login" text bein visible.

Reply:
Added in the manual [E2E wiki page](https://github.com/informatics-isi-edu/chaise/wiki/Manual-UX-and-E2E-Testing#login-link-for-404-error) as currently testing anonymous feature is not supported.